apps/prairielearn/src/pages/instructorAssessmentDownloads/instructorAssessmentDownloads.js

```diff
@@ -308,7 +308,7 @@ router.get(
         '',
       );
       const prefix = `${dirname}/`;
-      archive.append(null, { name: prefix });
+      archive.append('', { name: prefix });
```

Comment:
I just checked, and this call is unnecessary. Having it means the zip file will have an entry for the directory, but [the ZIP format](https://pkware.cachefly.net/webdocs/casestudies/APPNOTE.TXT) does not require separate entries for directories, unless some metadata is expected to be assigned to them (not the case here). Also, other directories created below based on paths that happen to contain a slash (e.g., if the QID has a path) don't have corresponding entries, and it still work. So the code here is correct, but could be removed altogether. Up to you.

Reply:
I think this is here to ensure that even if there aren't any files, the folder still appears when unzipping. I think that's a useful property to maintain for consistency?

Reply:
Good point.